<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEditName">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>52</y>
      <width>133</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSirname">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>114</y>
      <width>133</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditID">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>177</y>
      <width>133</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditType">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>133</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSubject">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>113</y>
      <width>133</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelName">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>19</y>
      <width>77</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя студента:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>82</y>
      <width>104</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Фамилия студента:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>144</y>
      <width>64</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID студента:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>18</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Тип сдачи:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Предмет:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMark">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>175</y>
      <width>133</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushSaveButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>143</y>
      <width>44</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Оценка:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushNewNodeButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Новая запись</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushSortIDButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Сортировка списка по ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushSortMarkButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Сортировка списка по оценкам</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushDeleteButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>300</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить студента</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineDeleteIDEdit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>300</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelName_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>280</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID для удаления</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushWatchButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Просмотр записей</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>493</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
